.github/workflows/ci.yaml

```diff
@@ -33,6 +33,11 @@ jobs:
           key: ${{ matrix.os }}-cargo-registry-${{ matrix.rust_version }}-${{ hashFiles('**/Cargo.lock') }}
           restore-keys: |
             ${{ matrix.os }}-cargo-registry-${{ matrix.rust_version }}-
+      - name: Add Gosub crate registry
+        run: |
+          echo "[registries.gosub]" >> ~/.cargo/config.toml
+          echo "index = \"sparse+https://registry.gosub.io/\"" >> ~/.cargo/config.toml
+          echo "" >> ~/.cargo/config.toml
```

Comment:
Can't you just directly specify the registry (url) in the Cargo.toml?

Reply:
i'm not sure actually..  i can try

Reply:
looks like you can't. You can only reference direct git repositories, not registry indexes.

Reply:
Ah, but you can move the `.cargo/config.toml` into the github repository, then it is way easier to run the browser